packages/frontend-v2/components/Plan/ScheduleTerm.tsx

```diff
@@ -32,14 +29,17 @@ interface ScheduleTermProps {
 
 export const ScheduleTerm: React.FC<ScheduleTermProps> = ({
   scheduleTerm,
-  isCourseInCurrPlan,
   addClassesToTermInCurrPlan,
   removeCourseFromTermInCurrPlan,
   isLastColumn,
 }) => {
   const { isOver, setNodeRef } = useDroppable({ id: scheduleTerm.id });
   const { isOpen, onOpen, onClose } = useDisclosure();
 
+  const isCourseInCurrTerm = (course: ScheduleCourse2<unknown>) => {
```

Comment:
As a clarification, is unknown being used as the generic here because we don't care about the type of the `id` property?

Reply:
precisely! this allows us to call the function with or without a type. and it also prevent the function itself from accessing the id type(without explicitly checking its type) because it doesn't know what it is.